FizzBuzz.py

```diff
@@ -1,27 +1,42 @@
-from types import ListType, IntType
+"""Print all integers from 1 to 100. However, for multiples of three,
+print 'Fizz' instead of the number, for multiples of five, print
+'Buzz', and for numbers which are multiples of both three and five,
+print 'FizzBuzz'.
+"""
+def is_appropriate(number):
+    """Assert whether or not a given number is an integer between 0 and
+    101. Is so, return the boolean True.
+    """
+    assert isinstance(number, int)
+    assert 0 < number <= 100
+    return True
```

Comment:
a function should never return just one half a bool - this is a code smell. if something returns true sometimes, it should return false sometimes other times.